django_mongodb/query.py

```diff
@@ -302,3 +305,50 @@ def register_nodes():
     Join.as_mql = join
     NothingNode.as_mql = NothingNode.as_sql
     WhereNode.as_mql = where_node
+
+
+class MongoQuerySet(QuerySet):
+    def raw_mql(self, raw_query, params=(), translations=None, using=None):
+        return MongoRawQuerySet(
+            raw_query,
+            model=self.model,
+            params=params,
+            translations=translations,
+            using=using,
+        )
+
+
+class MongoRawQuery(RawQuery):
+    def _execute_query(self):
+        connection = connections[self.using]
+        params_type = self.params_type
+        adapter = connection.ops.adapt_unknown_value
+        if params_type is tuple:
+            params = tuple(adapter(val) for val in self.params)
+        elif params_type is dict:
+            params = {key: adapter(val) for key, val in self.params.items()}
+        elif params_type is None:
+            params = None
+        else:
+            raise RuntimeError("Unexpected params type: %s" % params_type)
+
+        self.cursor = connection.cursor()
```

Comment:
I think you'll need to pass `model` to `MongoRawQuery` and then you can get the collection:
```
collection = connection.get_collection(model._meta.db_table)
collection.aggregate(self.sql)
```
It doesn't appear you need `MongoQuery` like I suggested before.